webpack/scenes/Subscriptions/SubscriptionsPage.js

```diff
@@ -69,7 +68,10 @@ class SubscriptionsPage extends Component {
       }
     }
 
-    if (!isEqual(organization, prevProps.organization)) {
+    // remove the loading attribute so the action isn't called when org starts loading
+    const { loading: _cloading, ...currentOrgInfo } = organization;
```

Comment:
I am removing the `loading` attribute from the object. to do this, I'm assigning it to a variable and pulling the rest of the object to another variable `currentOrgInfo`. If I just use `loading`, I can't use `loading` again in the next line since its a const, so I added 'c' and 'p' and then '_' to signify its not being used.

Definitely open to suggestions here, I don't love these names either :nauseated_face:

Reply:
Got it. Had to play in the repl but it makes sense. Very interesting syntax!

Reply:
what about:
```js
const getOrgInfo = (org) => {
  const { loading, ...info } = org;
  return info;
};

const currentOrgInfo = getOrgInfo(organization);
const prevOrgInfo = getOrgInfo(prevProps.organization);
```